awswrangler/s3/_read_parquet.py

```diff
@@ -201,23 +179,25 @@ def _read_parquet_metadata(
         ignore_empty=ignore_empty,
         s3_additional_kwargs=s3_additional_kwargs,
     )
+    version_ids = (
+        version_id if isinstance(version_id, dict) else {paths[0]: version_id} if isinstance(version_id, str) else None
+    )
 
     # Files
-    schemas: Tuple[Dict[str, str], ...] = _read_schemas_from_files(
+    schemas: List[pa.schema] = _read_schemas_from_files(
         paths=paths,
         sampling=sampling,
         use_threads=use_threads,
         boto3_session=boto3_session,
         s3_additional_kwargs=s3_additional_kwargs,
-        version_ids=version_id
-        if isinstance(version_id, dict)
-        else {paths[0]: version_id}
-        if isinstance(version_id, str)
-        else None,
-        ignore_null=ignore_null,
```

Comment:
This flag was added to address #1082 - is there a different way to do it now?

Reply:
It's still there, it's just that as I mention [above](https://github.com/awslabs/aws-data-wrangler/pull/1513#discussion_r946595253), the type casting has moved and is now centralised. So it's passed there instead

Reply:
Proof is the test you created in #1082 is still passing in this new version

Reply:
Ah ok thanks